parity-util-mem/src/malloc_size.rs

```diff
@@ -585,6 +585,7 @@ impl<T: MallocSizeOf> MallocSizeOf for parking_lot::RwLock<T> {
 ///
 /// and when no "any: "
 ///
+/// ```rust
 /// use parity_util_mem::{malloc_size, malloc_size_of_is_0};
 ///
 /// struct Data<T> { pub T }
```

Comment:
```suggestion
/// struct Data<T> { pub t: T }
```

Reply:
Oh I haven't realized it wasn't compiling before